exercises/practice/hello-world/hello_world_test.gd

```diff
@@ -0,0 +1,2 @@
+func test_hello_world(solution_script):
+	return ["Hello, World!", solution_script.hello()]
```

Comment:
Minor nitpick but the indent seems larger here than in hello_world.gd so one of them probably needs to be updated to match the other.

Reply:
Good point, I've missed this :+1: Since Godot editor uses tabs by default, I think we should use them too (consistently, in all files). I've already changed the files :)

Side note: I think that Godot will accept both types of indentation, as long as they are consistent within a file, but there is no need to confuse GDScript learners :P

Reply:
Okay, I think you just need to ping Erik to take a final look.

Reply:
It might be good to choose a formatter / linter if available to make sure everything is consistent between contributors and PRs. https://github.com/Scony/godot-gdscript-toolkit seems promising since it has both.  There's a GitHub Action for static analysis, but we could also use the CI for checking if a PR is well-formatted.

Reply:
I've added a new workflow with that toolkit. It does detect some errors (e.g. an unnecessary `return`) but it doesn't detect things like mixed indent (tabs vs spaces). Nevertheless, I think it's a good idea to have the linter already configured, as it might get better in the future, and it already helps a bit. Thanks for this suggestion! ;)